src/embedded-checkout/loading-indicator.ts

```diff
@@ -0,0 +1,120 @@
+import { LoadingIndicatorStyles } from './embedded-checkout-styles';
+
+const DEFAULT_STYLES: LoadingIndicatorStyles = {
+    size: 70,
+    color: '#d9d9d9',
+    backgroundColor: '#ffffff',
+};
+
+const ROTATION_ANIMATION = 'embedded-checkout-loading-indicator-rotation';
+
+export default class LoadingIndicator {
```

Comment:
I have a few doubts with the naming, maybe this could be LoadingOverlay? Indicator to me sounds like just the icon. 🍹 

`_container` could become `_overlay`

Reply:
I understand where you coming from. But in this case, the indicator is in fact the spinner icon that gets shown to the shopper when the page is initially loaded. So I think it is okay to keep it as "indicator"?